### DISCUSSION TITLE
Lasso Coefficients in R Markdown Output

### DISCUSSION THREAD

**Discussion by Sreenath1986** [ropensci/targets#947] [Help]
Hi, I am trying to load a 'lasso regression model' targets object in a R markdown (to be rendered as pdf). However, after rendering it is not printing the model coefficients. It just prints '#NULL' in the pdf for that part. However, if I run that code block/chunk inside the rmd file, it prints the coefficients properly in the console. 

`
```{r}
tar_load(lasso_model_csat)
coef(lasso_model_csat)
```
`
Instead if I call just the lasso model betas as below, it renders properly in the rmd's pdf output.

`
```{r}
tar_load(lasso_model_csat)
lasso_model_csat$beta
```
`
Please help me with this. Thank you.

**Comment by wlandau** [comment]
The pipeline saves each target in storage (by default, with `saveRDS()`). Not every R object can be saved this way and then loaded back into the R session: https://cran.r-project.org/web/packages/future/vignettes/future-4-non-exportable-objects.html. If you have an object like this, consider instead returning a saveable/exportable summary from the target, like a data frame from `broom::tidy()`.

Other than that, it's hard to identify the cause of the issue without a [reproducible example](https://stackoverflow.com/help/minimal-reproducible-example).
